mm2src/mm2_main/src/mm2.rs

```diff
@@ -403,7 +387,7 @@ pub fn mm2_main() {
     }
 
     if let Err(err) = run_lp_main(first_arg, &|_| ()) {
-        log!((err));
+        eprintln!("{}", err);
```

Comment:
If `MM_LOG` is set, would this be printed in file with old `log` macro?

Reply:
Done